File: 009.c
#include <stdio.h>
#include "common/euler.h"
void solution()
{
    int a, b, c;
    for (c = 1000; c > 2; c--)
        for (b = c - 1; b > 1; b--)
            for (a = b - 1; a > 0; a--)
                if (((a * a) + (b * b) == (c * c)) && ((a + b + c) == 1000))
                    printf("%d, %d, %d\n", a, b, c);
}

int main(int argc, char *argv[])
{
    euler_run(argv[0], solution);
    return 0;
}

File: 009.js
require('./common/euler').run(function () {
    for (var i = 1000; i > 2; i -= 1)
        for (var j = i - 1; j > 1; j -= 1)
            for (var k = j - 1; k > 0; k -= 1)
                if (((k * k) + (j * j) === (i * i)) && ((k + j + i) === 1000))
                    return k * j * i;
    return 'failed!';
});